packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/home/logs-inputs-home.html

```diff
@@ -1,26 +1,24 @@
 <div class="cui-page__content inputs-home">
-    <span class="oui-header_3" data-translate="inputs_title"></span>
+    <cui-page-content-title
```

Comment:
Could we use a UI-Kit component instead ? Or maybe just a heading ?

Reply:
This was already used on other pages as heading. My modification mainly consist to align usage on every page. For me it looks better with `cui-page-content-title` than with `oui-header_3`. If you have recommendation, tell me.

Reply:
@pdepaepe I agree that consistency should be key, but our latest UX/UI guidelines are brought with the UI-Kit and not the `cui-` components. 
To align usage on every page we should instead replace `cui-page-content-title` with headers where it's used not the other way round (uniformize with the latest guidelines not legacy ones :))

Reply:
Fixed